requirements.txt

```diff
@@ -9,7 +9,7 @@ black==19.3b0
 celery==4.4.2
 click==7.1.1
 decorator==4.4.2
-Django==3.0.5
+Django==3.0.7
```

Comment:
you can use requirements.in file for updating the packages, using pip-tools   -https://github.com/jazzband/pip-tools